pkg/clusterresource/builder.go

```diff
@@ -341,8 +344,8 @@ func (o *Builder) generateInstallConfigSecret() (*corev1.Secret, error) {
 			Namespace: o.Namespace,
 		},
 		Type: corev1.SecretTypeOpaque,
-		Data: map[string][]byte{
-			"install-config.yaml": d,
+		StringData: map[string]string{
```

Comment:
What is the purpose of this change?

Reply:
This allows us to create-cluster with -o yaml and the edit the install config without having to base64 decode the embedded secret data.

Reply:
Wonderful!